trade-module/src/main/java/io/edpn/backend/trade/configuration/EntityMapperConfig.java

```diff
@@ -1,11 +1,7 @@
 package io.edpn.backend.trade.configuration;
 
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.CommodityMapper;
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.CommodityMarketInfoMapper;
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.MarketDatumMapper;
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.RequestDataMessageMapper;
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.StationMapper;
-import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.SystemMapper;
+
+import io.edpn.backend.trade.infrastructure.persistence.mappers.entity.*;
```

Comment:
try not to use *-imports, this is likely an ID setting

![image](https://github.com/ed-pilots-network/backend/assets/1436380/97c4b4ec-32d8-43af-9e07-b505c1deca70)